helm/cloud-provider-cloud-director-app/charts/cloud-provider-for-cloud-director/templates/configmap.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vcloud-ccm-configmap
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+data:
+  vcloud-ccm-config.yaml: |+
+    vcd:
+      host: {{ .Values.cpiConfigmap.site }}
+      org: {{ .Values.cpiConfigmap.org }}
+      vdc: {{ .Values.cpiConfigmap.ovdc }}
+      vAppName: {{ .Values.cpiConfigmap.vAppName }}
+      network: {{ .Values.cpiConfigmap.ovdcNetwork }}
+      vipSubnet: {{ .Values.cpiConfigmap.vipSubnet }}

```

Comment:
For this part you can use helm `with` function :)
https://helm.sh/docs/chart_template_guide/control_structures/#modifying-scope-using-with